### DISCUSSION TITLE
Opacity for mesh surfaces

### DISCUSSION THREAD

**Discussion by jlvila** [jimy-byerley/pymadcad#30] [Q&A] [answered]
Dear All,

What is the best approach to have opacity on Mesh surfaces.
I tried with `fvec4(1.0,0.0,0.0,0.3)` into option (color=) with no effect.

thanks, 
jl

**Comment by jlvila** [comment, marked as answer]
It is a duplicate of an already asked question ;-)

**Reply by jimy-byerley** [reply]
yup ^^
